packages/protocol/contracts/stability/GrandaMento.sol

```diff
@@ -212,6 +392,44 @@ contract GrandaMento is
     return exchangeRate.multiply(adjustedSellAmount).fromFixed();
   }
 
+  /**
+   * @notice Gets the proposal identifiers of all exchange proposals in the
+   * Proposed or Approved state.
+   * @return An array of active exchange proposals IDs.
+   */
+  function getActiveProposalIds() external view returns (uint256[] memory) {
+    return activeProposalIds.getKeys();
+  }
+
+  /**
+   * @notice Gets the oracle CELO price quoted in the stable token.
+   * @dev Reverts if there is not a rate for the provided stable token.
+   * @param stableToken The stable token to get the oracle price for.
+   * @return The oracle CELO price quoted in the stable token.
+   */
+  function getOracleExchangeRate(address stableToken)
+    private
+    view
+    returns (FixidityLib.Fraction memory)
+  {
+    uint256 rateNumerator;
+    uint256 rateDenominator;
+    (rateNumerator, rateDenominator) = getSortedOracles().medianRate(stableToken);
+    // When rateDenominator is 0, it means there are no rates known to SortedOracles.
+    require(rateDenominator > 0, "No oracle rates present for token");
+    return FixidityLib.wrap(rateNumerator).divide(FixidityLib.wrap(rateDenominator));
+  }
+
+  /**
+   * @notice Sets the approver.
+   * @dev Sender must be owner.
+   * @param newApprover The new value for the spread.
+   */
+  function setApprover(address newApprover) public onlyOwner {
```

Comment:
Audit likely will flag approver can be the zero address, and I think that is on propose, so I'll document it in the natspec.

Reply:
(and add a test, wil cEUR we realized we couldn't set something as zero after the audit)